inifix/format.py

```diff
@@ -2,44 +2,86 @@
 import os
 import re
 import sys
+import warnings
 from typing import Optional
 from typing import Sequence
 
 from inifix.io import load
 
 NAME_COL_MINSIZE = 10
 COMMENT_COL = 80
+DEFAULT_PADDING_SIZE = 4
+
+
+def ljust(string: str, width: int) -> str:
+    """Similar to str.ljust, but will always append a whitespace ' ' at the end of the string, for the ini file to be usable"""
+    justified_name = string.ljust(width)
+    if (
+        len(string) < width
+    ):  # in this case, ljust already appended a whitespace : no need to append one
```

Comment:
Seems clear enough without a comment
```suggestion
    if len(string) < width:
```